### PR TITLE
Add ansifilter

### PR DESCRIPTION
**Pull request by asford**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by asford** [comment]
@conda-forge/help-c-cpp This is ready for review.

File: recipes/ansifilter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  number: 0
+  script: make && make install PREFIX=$PREFIX
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+test:
+  requires:
```
Comment (marcelotrevisani) [inline recipes/ansifilter/meta.yaml:24]:
```suggestion
```

File: recipes/ansifilter/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  commands:
+    - ansifilter --version | grep {{ version }}
```
Comment (marcelotrevisani) [inline recipes/ansifilter/meta.yaml:26]: I believe the ``grep`` here is not necessary
```suggestion
    - ansifilter --version
```

File: recipes/ansifilter/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://gitlab.com/saalen/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: make && make install PREFIX=$PREFIX
```
Comment (marcelotrevisani) [inline recipes/ansifilter/meta.yaml:16]:
```suggestion
  script:
    - make -j$CPU_COUNT
    - make install PREFIX=$PREFIX

File: recipes/ansifilter/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
+{% set name = "ansifilter" %}
+{% set version = "2.17" %}
+{% set sha256 = "5c8c4d86babbc952141341f70f1ac7a27de5f288b03938b7c8bcb5b84c4217ba" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (marcelotrevisani) [inline recipes/ansifilter/meta.yaml:10]: this line here is not necessary, if you want to, you can remove it
```suggestion
```

**Comment by asford** [comment]
Thank you for the review and close eye!

**Review by marcelotrevisani — APPROVED** [review]

**Pull request merged**